database/migrations/sql/20180603_create_initial_schema.sql

```diff
@@ -0,0 +1,461 @@
+);
+
+
+/*
+ * VISIT RELATED TABLES
+ */
+CREATE TABLE visit_activity_category (
+  visit_activity_category_id SERIAL NOT NULL UNIQUE,
+  category_name              VARCHAR NOT NULL,
+
+  CONSTRAINT visit_activity_category_pk PRIMARY KEY (visit_activity_category_id)
+);
+
+
+CREATE TABLE visit_activity (
+  visit_activity_id          SERIAL NOT NULL UNIQUE,
+  organisation_id            INT NOT NULL,
+  visit_activity_category_id INT NOT NULL,
+  activity_name              VARCHAR(80) NOT NULL,
+  monday                     BOOLEAN NOT NULL DEFAULT false,
+  tuesday                    BOOLEAN NOT NULL DEFAULT false,
+  wednesday                  BOOLEAN NOT NULL DEFAULT false,
+  thursday                   BOOLEAN NOT NULL DEFAULT false,
+  friday                     BOOLEAN NOT NULL DEFAULT false,
+  saturday                   BOOLEAN NOT NULL DEFAULT false,
+  sunday                     BOOLEAN NOT NULL DEFAULT false,
+  created_at                 TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+  modified_at                TIMESTAMP WITH TIME ZONE,
+  deleted_at                 TIMESTAMP WITH TIME ZONE,
+
+  CONSTRAINT visit_activity_pk                            PRIMARY KEY (visit_activity_id),
+  CONSTRAINT visit_activity_to_organisation_fk            FOREIGN KEY (organisation_id)            REFERENCES organisation ON DELETE CASCADE,
+  CONSTRAINT visit_activity_to_visit_activity_category_fk FOREIGN KEY (visit_activity_category_id) REFERENCES visit_activity_category
+);
+CREATE TRIGGER update_visit_activity_modified_at BEFORE UPDATE ON visit_activity
+  FOR EACH ROW EXECUTE PROCEDURE update_modified_at_column();
+
+
+CREATE TABLE visit (
+  visit_id          SERIAL NOT NULL UNIQUE,
+  user_account_id   INT NOT NULL,
+  visit_activity_id INT NOT NULL,
+  created_at        TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+  modified_at       TIMESTAMP WITH TIME ZONE,
+  deleted_at        TIMESTAMP WITH TIME ZONE,
+
+  CONSTRAINT visit_pk                   PRIMARY KEY (visit_id),
+  CONSTRAINT visit_to_user_fk           FOREIGN KEY (user_account_id)   REFERENCES user_account ON DELETE CASCADE,
+  CONSTRAINT visit_to_visit_activity_fk FOREIGN KEY (visit_activity_id) REFERENCES visit_activity
+);
+CREATE TRIGGER update_visit_modified_at BEFORE UPDATE ON visit
+  FOR EACH ROW EXECUTE PROCEDURE update_modified_at_column();
+
+
+CREATE TABLE visit_feedback (
+  visit_feedback_id SERIAL NOT NULL UNIQUE,
+  organisation_id   INT NOT NULL,
+  score             INT NOT NULL,
+  created_at        TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+  modified_at       TIMESTAMP WITH TIME ZONE,
+  deleted_at        TIMESTAMP WITH TIME ZONE,
+
+  CONSTRAINT visit_feedback_pk                 PRIMARY KEY (visit_feedback_id),
+  CONSTRAINT visit_feedback_to_organisation_fk FOREIGN KEY (organisation_id) REFERENCES organisation ON DELETE CASCADE,
+  CONSTRAINT visit_activity_valid_score        CHECK (score > -2 AND score < 2)
```

Comment:
Do we want a 5 point feedback scale?

Reply:
I don't know, do we?

Reply:
It's currently 3, we can check with Sonja.